docker/incremental_load.sh.tpl

```diff
@@ -22,38 +22,96 @@ RUNFILES="${PYTHON_RUNFILES:-${BASH_SOURCE[0]}.runfiles}"
 
```

Comment:
This script has overgrown in comlexity, I wonder if we should not replace it with a python script